handbook/engineering/releases/index.md

```diff
@@ -127,3 +127,8 @@ This is because:
 - We haven't built the automated testing and update infrastructure to make continuous customer releases reliable.
 
 In the future, we may introduce continuous releases if these issues become surmountable.
+
+### Weekend releases
+
+If the 20th falls on a non-working day, the release captain has the right but not the obligation to change the day (+/- 3 days) that the release images are published.
+If they decide to do this, they should move the google calendar events and announce the change in the #dev-announce channel.
```

Comment:
Maybe add some minimum time this announcement must be made just for clarity? Like 2 weeks ahead?

Reply:
We no longer announce releases in advance as part of the [single-day release](https://github.com/orgs/sourcegraph/projects/90) project.

If we guarantee that by the 20th the release is there, would there be a need for an announcement? Basically, if 20th falls on the weekend/holiday, we cut the last working day before that, so by the 20th, it's there and the blog can be sent the next working day after the 20th.